Sanya_form.HTML

```diff
@@ -0,0 +1,73 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width">
+  <title>Intro to HTML and Javascript</title>
+  <link href="style.css" rel="stylesheet" type="text/css" />
+</head>
+<body>
+   
+    <form>
+      <b><label>First Name:</label></b>
+      <input 
+            id="first_name" 
+            type="text" 
+            placeholder="Name"
+            maxlength="10" 
+            minlength="10"/>
+        
+        <br/>
+        
+      <b><label>Last Name:</label></b>
+      <input id="last_name" type="text" placeholder="Name" />
+        
+        <br/>
+
+      <b><label>Date_of_Birth:</label></b>
+      <input type="date" />
+
+        <br/>
+        
+      <b><label>Email:</label></b>
+      <input type="email" placeholder="email"/>
+        
+        <br/>
+        
+      <b><label>Gender:</label></b>
```

Comment:
Looks like this gender label is out of place. It should be next to the radio buttons. If you view the page in a browser, you will see that it is not in the right place